colorbleed/__init__.py

```diff
@@ -24,3 +24,8 @@ def uninstall():
     print("Deregistering global plug-ins..")
     pyblish.deregister_plugin_path(PUBLISH_PATH)
     avalon.deregister_plugin_path(avalon.Loader, LOAD_PATH)
+
+
+def register_loader_plugins():
```

Comment:
Is this needed for this particular PR? Seems like a remainder from something else?

Reply:
This is a remainder of the standalone plugins which we have discussed in PLN-105